### PR TITLE
[FIX] paymentInvoice creation and invoice status

### PR DESCRIPTION
**Pull request by Ando22**
## Description
There's an error in creating the payment invoice. the payload of the API needs to be validated before communicating to the DB. this impacts updating the payment status of the related invoice.

## Related Issues
This PR tackling this issue #271 

## Steps to Test
Create a Payment Invoice for an Invoice, based on Invoice Number. try to pay some of the item, to get `partial` status, and pay all the item to get `paid` of Invoice payment status

## Screenshots (if applicable)
Screen Shot 2023-09-08 at 18 00 02

## Checklist
- [x] I have tested these changes

### CODE REVIEW

File: frontend/.env [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_SERVER="https://dev-server.idurarapp.com"
\ No newline at end of file
+REACT_APP_BACKEND_SERVER="http://localhost:8888"
```
Comment (salahlalami) [inline frontend/.env:1]: revert env file

**Review by salahlalami — CHANGES_REQUESTED** [review]

**Comment by salahlalami** [comment]
@polymahh so it doesn't work

File: frontend/.env (hunk 1 above)
Comment (Ando22) [reply frontend/.env]: done

**Comment by polymahh** [comment]
@Ando22 there is no need for this PR if we are removing "add new payment"

**Pull request closed without merging**